pkg/resource/template/resource.go

```diff
@@ -75,7 +75,7 @@ func NewTemplateResource(path string, config Config) (*TemplateResource, error)
 		return nil, fmt.Errorf("Cannot process template resource %s - %s", path, err.Error())
 	}
 
-	tr := tc.TemplateResource
+	tr := &tc.TemplateResource
```

Comment:
This is changing the behavior.  This will now update the cached TemplateResource.

Reply:
hmm actually scrap that - tc is fresh each time.